core-bundle/src/Resources/contao/elements/ContentText.php

```diff
@@ -42,15 +46,14 @@ protected function compile()
 		$this->Template->addBefore = false;
 
 		// Add an image
-		if ($this->addImage && $this->singleSRC)
+		if ($this->addImage && null !== ($figureBuilder = $this->getFigureBuilderIfResourceExists($this->singleSRC)))
 		{
-			$objModel = FilesModel::findByUuid($this->singleSRC);
-
-			if ($objModel !== null && is_file(System::getContainer()->getParameter('kernel.project_dir') . '/' . $objModel->path))
-			{
-				$this->singleSRC = $objModel->path;
-				$this->addImageToTemplate($this->Template, $this->arrData, null, null, $objModel);
-			}
+			$figureBuilder
+				->setSize($this->size)
+				->setMetaData($this->objModel->getOverwriteMetaData())
+				->enableLightbox()
```

Comment:
Shouldn’t this be
```suggestion
				->enableLightbox($this->fullsize)
```
?

Reply:
:+1: Yes, looks like I missed that one.